### DISCUSSION TITLE
Experiences with Mennekes Amtrom EV Chargers?

### DISCUSSION THREAD

**Discussion by peterb99** [lbbrhzn/ocpp#1223] [Q&A]
Not a feature request, but don't know where else to ask.
Any experiences with integration of Mennekes Amtron chargers?
I have one, it seems to work, but curious if there are things I need to check?

Regards
Peter Brouwer

**Comment by andersbochr** [comment]
I just had the Mennekes Amtron 4You 560 C2 11kw installed. I can connect it but I if stop the charging I cannot start it again. I also cannot change the number of Amps. Which model do you have and did you have any of these problems?
